provision/site.yml

```diff
@@ -19,3 +19,8 @@
   hosts: localhost
   roles:
     - { role: 'output_writer'}
+
+- name: "inventory generation role"
+  hosts: localhost
+  roles:
+    - { role: 'inventory_gen'}
```

Comment:
Should this be also when state == "present"? Essentially, I didn't see anything that indicated the inventory_gen role should NOT run when state == "absent".